doc/controls/SafeArea.md

```diff
@@ -4,30 +4,35 @@
 
 The `ApplicationView.VisibleBounds` is the rectangular area of the screen which is completely unobscured by any window decoration, such as the status bar, rounded screen corners, or any type of screen notch.
 
-The `SafeArea` can also be used to ensure that the specified control will never be hidden by any sort of soft-input panel, such as the on-screen keyboard. This is done by observing the state of the keyboard and treating the area that it occupies when open as part of the "unsafe" area of the screen.
+`SafeArea` can also be used to specify certain areas of the UI that should adapt its layout in order to avoid being covered by any sort of soft-input panel, such as the on-screen keyboard. This is done by observing the state of the keyboard and treating the area that it occupies when open as part of the "unsafe" area of the screen.
 
 In some cases, it is acceptable for visible content to be partially obscured (a page background for example) and it should extend to fill the entire window. Other types of content should be restricted to the visible bounds (for instance: readable text, or interactive controls). `SafeArea` enables this kind of fine-grained control over responsiveness to the safe and "unsafe" areas of the screen.
 
 ## Properties
 
 ### Remarks
 
-All of the `SafeArea` properties can be used both as a dependency property or as an attached property, much like the `ScrollViewer` properties:
+`SafeArea` can be used as a `Panel` or as a set of attached properties on another `Panel`, much like the `ScrollViewer`:
```

Comment:
Sorry, I mean "panel" in the literal sense, like a container. An uwp `Panel` has a different meaning, which typically "_positions and arranges child objects_" (emphasis on the plural part).
Perhaps `ContentControl` is more appropriate in this context, also given that both SV and SA inherit from it.

Reply:
do a ctrl+f for `panel` and replace them with "control" or "container" whichever fits the best in its context.